app/javascript/app/components/data-explorer-content/data-explorer-content-styles.scss

```diff
@@ -54,3 +66,39 @@
     @include xy-gutters($gutter-position: 'top', $gutters: $gutter-padding * 3);
   }
 }
+
+.blank {
+  order: 2;
+}
+
+.paginate {
+  display: inline-block;
+  padding: 10px 15px;
+  text-align: center;
+  order: 1;
+
+  @media #{$tablet-landscape} {
+    order: 2;
+  }
+
+  a {
+    color: $dark-gray;
+    cursor: pointer;
+  }
+
+  li {
+    padding: 0 5px;
+    display: inline-block;
+  }
+}
+
+.active {
+  a {
+    color: $theme-secondary;
```

Comment:
I'm not being able to find the designs for the pagination, but would make sense to change the `$theme-secondary` colour for `$theme-color`?
Do not take this comment into account if you do not want, is just that I found the selected page number in the secondary colour too loud.

Reply:
Pagination may need some more space to avoid layout breaking in some cases (or allow less selectable pages)

![image](https://user-images.githubusercontent.com/6906348/42823841-df2d2c74-89de-11e8-8e9b-9571f6dbda17.png)